src/bootstrap-window.js

```diff
@@ -87,16 +95,27 @@
 
 		window['MonacoEnvironment'] = {};
 
-		// const baseUrl = sandbox ? // {{SQL CARBON EDIT}} Pending changes?
-		// 	`${bootstrapLib.fileUriFromPath(configuration.appRoot, { isWindows: safeProcess.platform === 'win32', scheme: 'vscode-file', fallbackAuthority: 'vscode-app' })}/out` :
-		// 	`${bootstrapLib.fileUriFromPath(configuration.appRoot, { isWindows: safeProcess.platform === 'win32' })}/out`;
+		const baseUrl = sandbox ?
```

Comment:
looks the same as the block above which are commented out.

Reply:
@alanrenmsft yes, the commented out code has been uncommented out and integrated into the subsequent logic.  Is there a question regarding that.

Reply:
my bad, for some reason I didn't realize the commented out code was removed. 🤣 , looks good